features/apim-analytics-feature/org.wso2.analytics.solutions.apim.analytics.feature/src/main/resources/siddhi-files/APIM_ALERT_CONFIGURATION.siddhi

```diff
@@ -53,6 +53,6 @@ on ApiCreatorAlertConfiguration.apiName == apiName and ApiCreatorAlertConfigurat
 
 from ApiSubscriberAlertConfigurationStream
 select *
-update or insert into ApiSubscriberAlertConfiguration
-set ApiSubscriberAlertConfiguration.thresholdRequestCountPerMin = thresholdRequestCountPerMin
-on ApiSubscriberAlertConfiguration.applicationId == applicationId and ApiSubscriberAlertConfiguration.apiName == apiName and ApiSubscriberAlertConfiguration.apiVersion == apiVersion;
\ No newline at end of file
+update or insert into ApiSubAlertConf
+set ApiSubAlertConf.thresholdRequestCountPerMin = thresholdRequestCountPerMin
+on ApiSubAlertConf.applicationId == applicationId and ApiSubAlertConf.apiName == apiName and ApiSubAlertConf.apiVersion == apiVersion;
```

Comment:
new line required

Reply:
added in 389b8295bf177fa698150dd87a9014ded5aa64a8 of https://github.com/wso2/analytics-apim as the file is moved to the above repo and renamed as APIM_CONFIGURATION_ALERT in analytics apim repo.